vehicle/raw_vehicle_cmd_converter/include/raw_vehicle_cmd_converter/node.hpp

```diff
@@ -52,8 +49,8 @@ class AccelMapConverter
   double max_throttle_;
```

Comment:
Out of curiosity: shouldn't we document all members? Here it's just half-half. And the above doc comments merely restate what the code does, whereas the meaning of `max_throttle` is not documented